packages/devextreme/js/ui/context_menu/ui.menu_base.js

```diff
@@ -400,36 +400,37 @@ class MenuBase extends HierarchicalCollectionWidget {
     }
 
     _renderItems(nodes, submenuContainer) {
-        if(nodes.length) {
-            this.hasIcons = false;
+        if(!nodes.length) {
```

Comment:
Unnecessary in this PR but code of this method will be changed in next PRs related to this feature

Reply:
Anyway it looks much better :) New option is just easier to read